src/bp/core/server.ts

```diff
@@ -288,6 +288,16 @@ export default class HTTPServer {
       }
     })
 
+    const handleBodyLimitError = (err, req, res, next) => {
+      if (err.statusCode === 413) {
+        this.logger
+          .error('You may need to increase httpServer.bodyLimit in file data/global/botpress.config.json')
+      }
+      next(err)
+    }
+
+    this.app.use(handleBodyLimitError)
```

Comment:
Can you add this in the handleErrors handler ? We could avoid adding an additional middleware.

Reply:
Yes, makes sense. Would it look better now?